src/components/dev-hub/byline-block.js

```diff
@@ -17,9 +21,21 @@ const AuthorLink = styled(Link)`
 `;
 
 const AuthorText = styled(P)`
+    display: inline-block;
     @media ${screenSize.upToLarge} {
         font-size: ${fontSize.xsmall};
     }
+    &:before {
```

Comment:
I'm concerned these pseudo-elements won't be picked up by search engines and screen readers. Can you add them directly to the text?